roborock/devices/mqtt_channel.py

```diff
@@ -95,6 +96,15 @@ async def _resolve_future_with_lock(self, message: RoborockMessage) -> None:
             else:
                 _LOGGER.debug("Received message with no waiting handler: request_id=%s", request_id)
 
+    async def send_message_no_wait(self, message: RoborockMessage) -> None:
```

Comment:
How about we just call this `publish`? i have a pending PR I have not sent yet to remove RPCs form this method and push it up a level, which is renaming this also.